src/methods/ReportTemplateGenerator.ts

```diff
@@ -0,0 +1,116 @@
+class ReportTemplateGenerator {
+  constructor(
+    public readonly filename: string = "",
+    public readonly template: string = ""
```

Comment:
What is your intention on making these props?

Reply:
I intended these properties to make the filename and LaTeX template customizable for users. Still, I will remove them because this improvement has to be implemented on the other pull requests.